README.md

```diff
@@ -54,275 +94,43 @@ See the [examples](https://github.com/GoogleCloudPlatform/gcp-service-broker/tre
-will be disabled. e.g.,
-
-```json
-{
-    "bt_plan": {
-        "name": "bt_plan",
-        "description": "Bigtable basic plan",
-        "storage_type": "HDD",
-        "num_nodes": "5",
-        "display_name": "Bigtable Plan",
-        "service": "b8e19880-ac58-42ef-b033-f7cd9c94d1fe"
-    }
-}
-```
-* `SPANNER_CUSTOM_PLANS` (A map of plan names to string maps with fields `guid`, `name`, `description`,
-`num_nodes` `display_name`, and `service` (Spanner's service id)) - if unset, the service
-will be disabled. e.g.,
-
-```json
-{
-    "spannerplan": {
-        "name": "spannerplan",
-        "description": "Basic Spanner plan",
-        "num_nodes": "15",
-        "display_name": "Spanner Plan",
-        "service": "51b3e27e-d323-49ce-8c5f-1211e6409e82"
-    }
-}
-```
-
-
-## Usage
-
-### As an App
+#### Optional env vars <a name="optional-env"></a>
 
-#### Update the manifest with ENV vars
-1. replace any blank variables that are in `manifest.yml` with your own ENV vars
+See [here](https://github.com/GoogleCloudPlatform/gcp-service-broker/blob/master/docs/installation.md#optional-env) 
+for instructions on providing database name and port overrides, ssl certs, and custom service plans for Cloud SQL, Bigtable, and Spanner.
 
-#### Push the service broker to CF and enable services
+#### Push the service broker to CF and enable services <a name="push"></a>
 1. `cf push gcp-service-broker`
 1. `cf create-service-broker <service broker name> <username> <password> <service broker url>`
 1. (for all applicable services, e.g.) `cf enable-service-access google-pubsub`
 
-### As a Tile
-
-#### Import the product into Ops Manager
-1. Click "Import a Product" and upload the .pivotal file from the product directory
-
-#### Add the product to your Dashboard
-1. Click the plus icon next to the uploaded product
+For more information, see the Cloud Foundry docs on [managing Service Brokers](https://docs.cloudfoundry.org/services/managing-service-brokers.html)
 
-#### Configure the Service Broker
-1. Click on the tile and fill in any required fields (tabs will be orange if updates are needed)
-1. Once the tile is green and updates are applied, review the service/plan access and
-update if necessary using cf disable-service-access. By default, all services and plans
-are enabled except CloudSQL, Bigtable, and Spanner (unless plans have been saved for them). If you wish to change this,
-you'll need to use the cf cli's service-access commands.
-
-### (Optional) Increase the default provision/bind timeout
+#### (Optional) Increase the default provision/bind timeout <a name="timeout"></a>
 It is advisable, if you want to use CloudSQL, to increase the default timeout for provision and
 bind operations to 90 seconds. CloudFoundry does not, at this point in time, support asynchronous
 binding, and CloudSQL bind operations may exceed 60 seconds. To change this setting, set
 `broker_client_timeout_seconds` = 90
```

Comment:
set this where? (if the answer is obvious to someone who regularly uses CF feel free to ignore this comment)